LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -27,6 +27,7 @@
     "css/cssom-view": "import", 
     "css/filter-effects": "import", 
     "css/geometry": "import", 
+    "css/mediaqueries": "import", 
```

Comment:
This is already below as `web-platform-tests/css/mediaqueries`. Can you please undo this change?

(This file should probably get cleaned up, but that's a different story)